app/views/companies/index.html.slim

```diff
@@ -0,0 +1,11 @@
+.container
+  .section
+    - @companies.group_by(&:supporter?).reverse_each do |supporter, companies|
+      - if supporter
+        h2.section-heading We #{icon :heart} Sharing
+        p.text-center These #{companies.size} companies have sponsored at least two local meetups this year with venue, food, or talks.
```

Comment:
Yeah. This is the simplest possible that works. An idea could be to make it "in the last 12 months", but that would require us to store the date of each contribution. Something simpler could be to make `contribution_count` a hash, indexed by the year.